Rakefile

```diff
@@ -38,6 +38,14 @@ namespace :smithy do
   end
 
   task 'spec' => %w[spec:unit spec:endpoints]
+
+  desc 'Convert all fixture smithy models to JSON AST representation'
+  task 'sync-fixtures' do
```

Comment:
I'm fine with sync, but we could consider a validation task instead (one that doesn't overwrite but just checks)